model/users.go

```diff
@@ -0,0 +1,30 @@
+package model
+
+import "time"
+
+type User struct {
+	UserID         int       `json:"user_id"`
+	GithubUsername string    `json:"github_username"`
+	GithubID       int       `json:"github_id"`
+	CreatedAt      time.Time `json:"created_at"`
+	DeletedAt      time.Time `json:"deleted_at"`
+	UpdatedAt      time.Time `json:"updated_at"`
```

Comment:
Do we need these fields in the JSON response? I personally don't think it's necessary. There is coupling between the struct that used to access the database (the "model" in the MVC pattern, which requires these fields) as well as the struct that is used to create the JSON response (the "view" in MVC, which do not require these fields). Ideally, they should be separate structs.